### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@SharS @roykuntz
Updating this to reflect correct information for phone number attributes that can be used for Auto Attendant Extension Dialing.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -218,7 +218,12 @@ If you select **Disconnect**, the caller is disconnected after the greeting play
...
+>- Users you wish to make available for Dial By Extension need to have an extension specified as part of one of the following phone attributes defined in Active Directory or Azure Active Directory [Microsoft 365 admin center](https://docs.microsoft.com/office365/admin/add-users/add-users?view=o365-worldwide#use-the-new-admin-center-to-add-users).
+>    - HomePhone
+>    - Mobile/MobilePhone
+>    - TelephoneNumber/PhoneNumber
+>    - OtherTelephone
+>- The required format to enter the extension in the user phone number field is is either `+<phonenumber>;ext=<extension>` or `x<extension>`.
 >- Assigning an extension in Teams Admin center is not currently supported. You must either use the [Set-MsolUser](https://docs.microsoft.com/powershell/module/msonline/set-msoluser?view=azureadps-1.0) powershell command or the Microsoft 365 admin center.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md]:
```suggestion
>- Assigning an extension in Teams Admin center is not currently supported. You must either use the [Set-MsolUser](https://docs.microsoft.com/powershell/module/msonline/set-msoluser?view=azureadps-1.0) PowerShell command or the Microsoft 365 admin center.
```

**Comment by scanum** [comment]
@kenwith th PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kenwith** [comment]
@scanum - it looks like there is still an open copy edit suggestion. It needs to be addressed before I merge otherwise the copy edit will be wasted.

@carolynblanding - would you please see copy edit above and merge it if you agree? Thanks.

**Pull request merged**